docs/php/versioning.md

```diff
@@ -95,7 +95,7 @@ if ($v === 1) {
 
 You'll note that `minSupported` has changed from `DEFAULT_VERSION` to `1`. If an older version of the
 Workflow Execution history is replayed on this code, it will fail because the minimum expected version
```

Comment:
```suggestion
You'll note that `minSupported` has changed from `DEFAULT_VERSION` to `1`.
If an older version of the Workflow Execution history is replayed on this code, it fails because the minimum expected version is 1.
```